internal/app/connectconformance/testsuites/data/connect_with_get.yaml

```diff
@@ -2,14 +2,15 @@ name: Connect with GET
 relevantProtocols:
   - PROTOCOL_CONNECT
 relevantCompressions:
-  # TODO - Clients only use compression for GET requests if the request is
-  # larger than a particular size. So since we aren't sending a request here
-  # that exceeds that, our requests won't use compression even if we specify
-  # it. So, we're limiting this test to identity
+  # TODO: The reference client only uses compression for GET requests if the
+  # request is larger than a particular size. So since we aren't sending a
+  # request here that exceeds that, our requests won't use compression even
+  # if we specify it. So, we're limiting this test to identity.
```

Comment:
Sorry for my compulsion to tweak stuff 😂

I don't actually know how other clients behave, but connect-kotlin behaves differently. So I thought it would be useful to clarify _which_ client behaves this way.

The fact that implementations are not consistent on how to enable compression in GET requests does make it hard to test this case. FWIW, I think the reason a client might behave like this is because the compression output may not be canonical/deterministic, which means things like caching by URL don't work well. So they only compress when necessary to fit the request into a "reasonably-sized" URL.